math/smatrix/include/algebra/math/impl/smatrix_matrix.hpp

```diff
@@ -0,0 +1,63 @@
+/** Algebra plugins library, part of the ACTS project
+ *
+ * (c) 2020-2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s).
+#include "algebra/qualifiers.hpp"
+
+// ROOT/Smatrix include(s).
+#include <Math/SMatrix.h>
+
+namespace algebra::smatrix::matrix {
+
+/// "Matrix actor", assuming an Eigen matrix
+template <typename scalar_t>
+struct actor {
```

Comment:
Yes I changed its name to `matrix::actor`.  and we don't have to worry about the convention because it's in `matrix` namespace. 
In detray side, it will be defined as: 
```c++
using matrix_actor = matrix::actor;
```

Reply:
Sure, I just meant, let's not name it ```matrix_actor```, since an actor is something different in detray. But we can of course do that on the detray side.